### PR TITLE
Forbid execution under PHP 8.0.0

### PR DESCRIPTION
**Pull request by keradus**
To be merged when PHP 8.0.1 is released

ref https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5399#discussion_r549422178
> PHP 8.0.1 is scheduled to be released on January 7th.

### CODE REVIEW

**Comment by shouze** [comment]
[Here we are! :tada:](https://www.php.net/ChangeLog-8.php#8.0.1)

**Comment by keradus** [comment]
GitHub CI is still having PHP 8.0.0 https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5415/checks?check_run_id=1664321299#step:10:12

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
             execute-migration-rules: 'yes' # should be checked on highest supported PHP version
 
           - operating-system: 'ubuntu-20.04'
-            php-version: '8.0'
+            php-version: '8.0.1'
```
Comment (keradus) [inline .github/workflows/ci.yml:49]:
```suggestion
            php-version: '8.0'
```

not working

File: php-cs-fixer [hunk 2]
```diff
@@ -25,6 +25,21 @@ if (defined('HHVM_VERSION_ID')) {
...
 } elseif (!defined('PHP_VERSION_ID') || \PHP_VERSION_ID < 50600 || \PHP_VERSION_ID >= 70500) {
     fwrite(STDERR, "PHP needs to be a minimum version of PHP 5.6.0 and maximum version of PHP 7.4.*.\n");
+    fwrite(STDERR, 'Current PHP version: '.PHP_VERSION.".\n");
+
+    if (defined('PHP_VERSION_ID') || \PHP_VERSION_ID === 80000) {
+        fwrite(STDERR, "PHP CS Fixer is not able run on PHP 8.0.0 due to bug in PHP tokenizer (https://bugs.php.net/bug.php?id=80462).\n");
+        fwrite(STDERR, "Update PHP version to unblock execution.\n");
+
+        exit(1);
+    }
```
Comment (keradus) [inline php-cs-fixer:35]:
```suggestion
```

falsy conflict fix

**Pull request merged**